code/Aggregate.m

```diff
@@ -0,0 +1,25 @@
+function Aggregate(genefile,Data,outdir)
```

Comment:
We discussed possibly renaming this `SINGE_Aggregate`

We should comment the arguments so that users know what input is expected.